### DISCUSSION TITLE
Integrate with MS Teams

### DISCUSSION THREAD

**Discussion by Ayanmullick** [community/community#3141] [Discussions] [answered]
Voice discussions can be even more engaging. Teams integration could enable the [discussions' transcript](https://docs.microsoft.com/en-us/microsoftteams/cloud-recording#turn-on-or-turn-off-recording-transcription) to be posted on GitHub which could then improve the `Time to answer` for a discussion.

**Comment by evi-liu** [comment, marked as answer]
Thanks for the feedback! We have webhooks on our roadmap, which should hopefully enable integrations with MS Teams!
